Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ElasticWindowMultiple.py

```diff
@@ -19,15 +21,23 @@ def _normalize_by_index(workspace, index):
     @param workspace    The workspace to normalize.
     @param index        The index of the y-value to normalize by.
     """
+    number_of_histograms = workspace.getNumberHistograms()
+
+    for idx in range(0, number_of_histograms):
+        y_values = workspace.readY(idx)
+        y_errors = workspace.readE(idx)
+
+        # Normalise y values
+        scale = np.reciprocal(y_values[index])
```

Comment:
Is it possible for these y values to be 0? If so, may be worth adding some small value (1e-8) before taking reciprocal to avoid RuntimeErrors